.github/COMMIT_TEMPLATE

```diff
@@ -0,0 +1,3 @@
+Jira issue: {{ key }}
+{% if parent__key %}Jira story: {{ parent__key }}{% endif %}
+{% if summary %}Jira summary: {{ summary }}{% endif %}
```

Comment:
How there is "Jira information" above this I like to suggest other template

```
Task ID: {{ key }}
{% if parent__key %}Story ID: {{ parent__key }}{% endif %}
{% if summary %}Summary: {{ summary }}{% endif %} 
```

Reply:
See: 56debed